server/common/default_config.py

```diff
@@ -14,6 +14,20 @@
     server_timing_headers: false
     csp_directives: null
 
+    # By default, cellxgene will serve backend api requests from the same base url as the frontend webpage.
+    # In general backend_base_url and frontend_base_url will not need to be set.
+    # There are two reasons to set these parameters:
+    #  1. Oauth authentication is used; the oauth server will redirect back to the backend_base_url after login,
+    #     which then redirects back to the frontend_base_url.  If the frontend_base_url is not set, it will default to
+    #     the backend_base_url.  If oauth authentication is used, the backend_base_url must be set.
+    #     For a local test (where the server runs on "http://localhost:<port>"), then the backend_base_url may be
+    #     set to the string "local".
+    #  2. The cellxgene deploymnent is in an environment where the frontend and backend have
+    #     different base urls.  In this case both backend_base_url and frontend_base_url must be set.
+    #     It is up to the server admin to ensure that the networking is setup correctly for this environment.
+    backend_base_url: null
```

Comment:
How do you feel about renaming this `api_base_url` and `web_base_url`? I think it can get a little confusing since the frontend_base_url is _technically_ still backend. Having `api` in what you now name `backend_base_url` makes it much more clear what it is actually serving.

Reply:
Sure, sounds good.